expression/builtin_control_vec_generated.go

```diff
@@ -16,27 +16,860 @@
 package expression
 
 import (
+	"time"
+
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
 )
 
-func (b *builtinIfIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
+func (b *builtinCaseWhenIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
 	n := input.NumRows()
-	buf0, err := b.bufAllocator.get(types.ETInt, n)
+	args, l := b.getArgs(), len(b.getArgs())
+	whens := make([]*chunk.Column, l/2)
+	whensSlice := make([][]int64, l/2)
+	thens := make([]*chunk.Column, l/2)
+	var eLse *chunk.Column
+
+	thensSlice := make([][]int64, l/2)
+	var eLseSlice []int64
+
+	for j := 0; j < l-1; j += 2 {
+		bufWhen, err := b.bufAllocator.get(types.ETInt, n)
+		if err != nil {
+			return err
+		}
+		defer b.bufAllocator.put(bufWhen)
+		if err := args[j].VecEvalInt(b.ctx, input, bufWhen); err != nil {
+			return err
+		}
+		whens[j/2] = bufWhen
+		whensSlice[j/2] = bufWhen.Int64s()
+
+		bufThen, err := b.bufAllocator.get(types.ETInt, n)
+		if err != nil {
+			return err
+		}
+		defer b.bufAllocator.put(bufThen)
+		if err := args[j+1].VecEvalInt(b.ctx, input, bufThen); err != nil {
+			return err
+		}
+		thens[j/2] = bufThen
+
```

Comment:
How about removing this unnecessary blank line.

Reply:
It will make the generator code more ugly. We should keep the code in generator clean, but not the generated code.